packages/core/lib/patchers/aws3_p.ts

```diff
@@ -40,6 +40,7 @@ const buildAttributesFromMetadata = async (
   service: string,
   operation: string,
   region: string,
+  commandInput: any,
```

Comment:
Is it possible to modify this to account for the possibility of the `Command` not having an `input` property?

Reply:
AFAIK all the commands from AWS SDK v3 inherits from the abstract class `Command` from internal package [`smithy-client`](https://github.com/aws/aws-sdk-js-v3/blob/main/packages/smithy-client/src/command.ts). Meaning there shouldn't be a case when the `Command` doesn't have an `input` prop.

But, ofc, we can be a bit more defensive – introduce a few conditions and fallback to a previous behavior.